### PR TITLE
 Added explanation for the --s <SETTINGS> flag 

### LINKED ISSUE #11589: Specify --s <SETTINGS FILE> settings file format
I suppose it's like a runsettings file, but it's not obvious.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 61ca4064-7481-a263-60fc-c70127f2ad16
* Version Independent ID: 2878af44-8208-7bd8-46d4-3acda3e82ce0
* Content: [dotnet test command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test?tabs=netcore21)
* Content Source: [docs/core/tools/dotnet-test.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-test.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by roel4ez**
## Summary

Added an explanation for the `--s <SETTINGS>` flag, and clarified with a link to the usage of such a file.

Fixes #11589 (if available)

### CODE REVIEW

File: docs/core/tools/dotnet-test.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ The directory where the test results are going to be placed. If the specified di
 
 `-s|--settings <SETTINGS_FILE>`
 
-Settings to use when running tests.
+The `.runsettings` file to use for running the tests. [Configure unit tests by using a .runsettings file](https://docs.microsoft.com/en-us/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2019)
```
Comment (Thraka) [inline docs/core/tools/dotnet-test.md]:
```suggestion
The `.runsettings` file to use for running the tests. [Configure unit tests by using a `.runsettings` file.](https://docs.microsoft.com/en-us/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2019)
```

File: docs/core/tools/dotnet-test.md [hunk 2]
```diff
@@ -184,7 +184,7 @@ The directory where the test results are going to be placed. If the specified di
...
```
Comment (Thraka) [inline docs/core/tools/dotnet-test.md]:
```suggestion
The `.runsettings` file to use for running the tests. [Configure unit tests by using a `.runsettings` file.](https://docs.microsoft.com/en-us/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2019)
```

File: docs/core/tools/dotnet-test.md [hunk 3]
```diff
@@ -234,7 +234,7 @@ Directory in which to find the binaries to run.
...
```
Comment (Thraka) [inline docs/core/tools/dotnet-test.md]:
```suggestion
The `.runsettings` file to use for running the tests. [Configure unit tests by using a `.runsettings` file.](https://docs.microsoft.com/en-us/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2019)
```

**Review by Thraka — CHANGES_REQUESTED** [review]
Thanks a bunch for doing this! It helps a lot. 

Some minor suggestions for you to approve. I added a period at the end of the sentence-links and I also code-blocked the file reference in the link so it matches the style of the normal text to the left of it.

Thanks again!

**Comment by roel4ez** [comment]
Approved the edits, thanks!

File: docs/core/tools/dotnet-test.md [hunk 4]
```diff
@@ -114,7 +114,7 @@ The directory where the test results are going to be placed. If the specified di
 
 `-s|--settings <SETTINGS_FILE>`
 
-Settings to use when running tests.
+The `.runsettings` file to use for running the tests. [Configure unit tests by using a `.runsettings` file.](https://docs.microsoft.com/en-us/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2019)
```
Comment (Thraka) [inline docs/core/tools/dotnet-test.md]:
```suggestion
The `.runsettings` file to use for running the tests. [Configure unit tests by using a `.runsettings` file.](/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2019)
```

Comment (Thraka) [inline docs/core/tools/dotnet-test.md]:
```suggestion
The `.runsettings` file to use for running the tests. [Configure unit tests by using a `.runsettings` file.](/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2019)
```

File: docs/core/tools/dotnet-test.md [hunk 5]
```diff
@@ -184,7 +184,7 @@ The directory where the test results are going to be placed. If the specified di
...
```
Comment (Thraka) [inline docs/core/tools/dotnet-test.md]:
```suggestion
The `.runsettings` file to use for running the tests. [Configure unit tests by using a `.runsettings` file.](/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2019)
```

File: docs/core/tools/dotnet-test.md [hunk 6]
```diff
@@ -234,7 +234,7 @@ Directory in which to find the binaries to run.
...
```
Comment (Thraka) [inline docs/core/tools/dotnet-test.md]:
```suggestion
The `.runsettings` file to use for running the tests. [Configure unit tests by using a `.runsettings` file.](/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2019)
```

**Review by Thraka — CHANGES_REQUESTED** [review]
@roel4ez I totally missed this the first visual pass, sorry. The links to visual studio have a locale in them, the en-us. I'll edit those and I think we can link directly to the file instead of the published link. 

I *think* this is how it should be formatted. Once you commit these it will kick off a build and we can test. Sorry about that.

**Comment by roel4ez** [comment]
ok thanks for noticing. The build says it's ok!

**Comment by Thraka** [comment]
Awesome! Thanks again 👍

**Review by Thraka — APPROVED** [review]

**Pull request merged**